src/pkg/artifact/candidate.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package art
+package artifact
```

Comment:
According to the "single responsibility principle" for pkg, please move the selector related code into a separated package "src/pkg/selector"